queue_services/entity-filer/src/entity_filer/filing_processors/special_resolution.py

```diff
@@ -0,0 +1,43 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for Special Resolution filings."""
+from typing import Dict
+
+from dateutil.parser import parse
+from entity_queue_common.service_utils import logger
+from legal_api.models import Business, Filing, Party, Resolution
+
+
+def process(business: Business, filing: Dict, filing_rec: Filing):
+    """Render the special resolution filing unto the model objects."""
+    logger.debug('Processing Special Resolution : %s', filing)
+    if(resolution_filing := filing.get('specialResolution')):
+        resolution = Resolution(
+            resolution=resolution_filing.get('resolution'),
+            resolution_type=Resolution.ResolutionType.SPECIAL.value,
+            resolution_sub_type=filing_rec.filing_type
+        )
+
+        if(signatory := resolution_filing.get('signatory')):
+            party = Party(
```

Comment:
did we want to include a value for `party_type`?  maybe `person` or `signatory`?

Reply:
party_type = db.Column('party_type', db.String(30), default=PartyTypes.PERSON.value). Default is person.

Reply:
ah ic.  👍